src/mame/mr/dribling.cpp

```diff
@@ -451,11 +484,35 @@ void dribling_state::dribling(machine_config &config)
 	screen.set_palette("palette");
 
 	PALETTE(config, "palette", FUNC(dribling_state::palette), 256);
-
 	// sound hardware
-}
+	SPEAKER(config, "mono").front_center();
```

Comment:
You’ve lost the blank line between blocks here.

Reply:
Done